docs/index.html

```diff
@@ -4,6 +4,8 @@
 const urls = {
   '':                 'https://mozilla.org',
   'firefox':          'https://firefox.com/',
+  'firefox/android':  'https://www.mozilla.org/firefox/android/',
+  'firefox/nightly':  'https://www.mozilla.org/firefox/channel/desktop/#nightly',
```

Comment:
If you use nightly.mozilla.org from android it will redirect you to the android page.